speculative_decoding/config.yaml

```diff
@@ -1,28 +1,34 @@
 base_image:
-  image: nvcr.io/nvidia/tritonserver:23.10-trtllm-python-py3
+  image: nvcr.io/nvidia/tritonserver:24.01-trtllm-python-py3
   python_executable_path: /usr/bin/python3
+environment_variables: {}
+external_package_dirs: []
 model_metadata:
   engine_repository: baseten/specdec-target-mistral-7B
   speculative_decoding:
     draft_engine_repository: baseten/specdec-draft-gpt2
     draft_tokenizer_repository: gpt2
     max_num_draft_tokens: 4
   tags:
-    - text-generation
+  - text-generation
   tensor_parallelism: 1
   tokenizer_repository: mistralai/Mistral-7B-v0.1
+model_name: specdec
 python_version: py311
 requirements:
-  - colorama
-  - csv
-  - huggingface_hub
-  - numpy
-  - pandas
-  - pydantic
-  - transformers
-  - tritonclient[all]
+- colorama
+- huggingface_hub
+- numpy
+- pandas
+- pydantic
+- transformers
+- tritonclient[all]
 resources:
   accelerator: A100
+  cpu: '1'
```

Comment:
Maybe remove these. A100 automatically comes with good defaults, these low numbers may be problematic and right sizing is not straightforward.